packages/react-tiles/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@datacommonsorg/react-tiles",
+  "version": "1.0.0",
```

Comment:
nit: change to 0.0.1 (let's bump to 1.x when it's production-ready)

Reply:
Done!